ddl/reorg.go

```diff
@@ -80,14 +83,14 @@ func (d *ddl) runReorgJob(t *meta.Meta, job *model.Job, f func() error) error {
 		}()
 	}
 
-	waitTimeout := waitReorgTimeout
+	waitTimeout := defaultWaitReorgTimeout
 	// if d.lease is 0, we are using a local storage,
 	// and we can wait the reorganization to be done here.
 	// if d.lease > 0, we don't need to wait here because
-	// we will wait 2 * lease outer and try checking again,
+	// we should update some job's progress context and try checking again,
 	// so we use a very little timeout here.
 	if d.lease > 0 {
-		waitTimeout = 50 * time.Millisecond
+		waitTimeout = ReorgWaitTimeout
```

Comment:
Why enlarge this from 50ms to 1s?

Reply:
If waitTimeout is 50ms, tidb will print 60 lines log per second. It's disturbed and useless.

Reply:
when we encounter a reorgWaitTimeout, three log will print like below:
```
2018/03/20 15:30:16.642 reorg.go:116: [info] [ddl] run reorg job wait timeout 1s, handled 11474 rows, current done handle 2110505137495968, err <nil>
2018/03/20 15:30:16.653 ddl_worker.go:378: [info] [ddl] schema version doesn't change
2018/03/20 15:30:16.655 ddl_worker.go:277: [info] [ddl] run DDL job ID:48, Type:add index, State:running, SchemaState:write reorganization, SchemaID:1, TableID:29, RowCount:11474, ArgLen:0, start time: 2018-03-20 15:29:51.881 +0800 CST, Err:<nil>, ErrCount:0, SnapshotVersion:398860220388081666
```